src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -9,7 +9,11 @@ public static double hundredMsToMin(double ms) {
         return hundredMsToSec(ms) * 60;
     }
 
-    public static double hundredMsToTickPerMin(double ms) {
-        return hundredMsToMin(ms) * 2048;
+    public static double ticksToRevolutions(double ticks) {
+        return ticks / 4048;
+    }
+
+    public static double ticksPerHundredMsToRPM(double velocityInHundredMs) {
```

Comment:
Change parameter name to `velocityInTicksPerHundredMs`. HundredMs is a measurement of time not velocity

Reply:
you can write 100ms
and you don't have to write "velocity".
it can just be `ticksPer100Ms`

Reply:
yes